README.md

```diff
@@ -1 +1,115 @@
-# Ruby-TDD
\ No newline at end of file
+# TDD
+# 📗 Table of Contents
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+# 📖 TDD <a name="about-project"></a>
+> **TDD** This application is designed to help the librarian manage the library's collection of books and keep track of who borrows them. It provides an organized and efficient way to add students, teachers, and books, and maintain records of borrowings
+## 🛠 Built With <a name="built-with"></a>
+> This project is built with Ruby.
+<details>
+  <summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby - A Programmer's Best Friend</a></li>
+  </ul>
+</details>
+
+## Key Features <a name="key-features"></a>
+> - **Add new Book**
+> - **Create Teacher**
+> - **Create Student**
+> - **Save Records of who Borrowed Books**
```

Comment:
- [ ] Kindly add key features of what you did inside this project, not for the old project.